pkg/s3utils/utils.go

```diff
@@ -170,7 +170,7 @@ func IsAmazonFIPSGovCloudEndpoint(endpointURL url.URL) bool {
 	if endpointURL == sentinelURL {
 		return false
 	}
-	return endpointURL.Host == "s3-fips-us-gov-west-1.amazonaws.com" ||
+	return endpointURL.Host == "s3-fips.us-gov-west-1.amazonaws.com" ||
```

Comment:
You should add this as another hostname, don't break what already exists

Reply:
@harshavardhana Is it needed? Just wondering how this would be working for anyone?

Reply:
Alright, I made the modifications 👍🏼